mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS kp net/minecraft/util/Identifier
-	FIELD a owner Ljava/lang/String;
+	FIELD a namespace Ljava/lang/String;
```

Comment:
I'm still not sure about it. I recommend you remove it from here and open up an issue.